src/js/components/pages.js

```diff
@@ -348,3 +352,53 @@ export class Service {
 }
 
 export class ServicePointPage extends Service {}
+
+export class NoticeIndex {
+  constructor(content) {
+    this.breadcrumbItems = getBreadcrumbsWithLabel(content.ancestor_pages);
+    this.childPages = content.child_pages;
+  }
+
+  renderChildPageLinks() {
+    const childPageLinks = this.childPages.map((page) => {
+      const props = {
+        title: page.title,
+        url: page.url,
+        subtitle: "",
```

Comment:
subtitle was supposed to be the publication date, according to the mockup, wasn't it?